Hernan-Alonso/Drinks_Finder
# [FEAT] - Drink recipe finder

### **What was done:**
- Drink recipe finder. The basics of this proyect is to search for a drink and a style of that drink, to show a list of recipes available. If the user clicks the CTA of a specific drink, the information of it will show up in a modal.

### **Data needed to test:**
- Drink beverages common in saxon language, such as tequila, vodka. Fernet won't do the trick.

### **How to test:**
1. Search a drink beverage with a category and see result. To see the recipe of the drink click in CTA "View recipe", this should list the ingredients a measurements of the drink.

### **What is missing:**
1. Other APIs call.
